build.sh

```diff
@@ -0,0 +1,177 @@
+    git clone --depth 1 --branch ${ov_branch} ${ov_repo} ${ivsr_ov_dir}
+    git config --global --add safe.directory ${ivsr_ov_dir}
+  fi
+  cd ${ivsr_ov_dir}
+  #git checkout ${OV_BRANCH}
+  git submodule update --init --recursive
+
+  ## applying ov22.3 patches to enable Enhanced BasicVSR model
+  for patch_file in $(find ../patches -iname "*.patch" | sort -n);do
+      echo "Applying: ${patch_file}"
+      git am --whitespace=fix ${patch_file}
+  done
+
+  mkdir -p build && cd build && \
+  cmake \
+    -DCMAKE_INSTALL_PREFIX=${PWD}/../install \
+    -DENABLE_INTEL_CPU=ON \
+    -DENABLE_CLDNN=ON \
+    -DENABLE_INTEL_GPU=ON \
+    -DENABLE_ONEDNN_FOR_GPU=OFF \
+    -DENABLE_INTEL_GNA=OFF \
+    -DENABLE_INTEL_MYRIAD_COMMON=OFF \
+    -DENABLE_INTEL_MYRIAD=OFF \
+    -DENABLE_PYTHON=ON \
+    -DENABLE_OPENCV=ON \
+    -DENABLE_SAMPLES=ON \
+    -DENABLE_CPPLINT=OFF \
+    -DTREAT_WARNING_AS_ERROR=OFF \
+    -DENABLE_TESTS=OFF \
+    -DENABLE_GAPI_TESTS=OFF \
+    -DENABLE_BEH_TESTS=OFF \
+    -DENABLE_FUNCTIONAL_TESTS=OFF \
+    -DENABLE_OV_CORE_UNIT_TESTS=OFF \
+    -DENABLE_OV_CORE_BACKEND_UNIT_TESTS=OFF \
+    -DENABLE_DEBUG_CAPS=ON \
+    -DENABLE_GPU_DEBUG_CAPS=ON \
+    -DENABLE_CPU_DEBUG_CAPS=ON \
+    -DCMAKE_BUILD_TYPE=Release \
+    .. && \
+  make -j $(nproc --all) && \
+  make install
+  echo "Build OpenVINO finished."
+    
+  source ${PWD}/../install/setupvars.sh
+}
+
+build_install_ivsr_sdk() {
+  echo "Building and installing iVSR SDK..."
+
+  ivsr_sdk_dir=${base_dir}/ivsr_sdk/
+  cd ${ivsr_sdk_dir}
+  mkdir -p build && cd build && cmake \
+    -DENABLE_LOG=OFF -DENABLE_PERF=OFF -DENABLE_THREADPROCESS=ON \
+    -DCMAKE_BUILD_TYPE=Release .. && \
+  make -j $(nproc --all)
+  sudo make install
+  echo "Build ivsr sdk finished."
+}
+
+build_ffmpeg() {
+  echo "Building FFMPEG with specific libraries support..."
+  sudo -E apt-get update && \
+    DEBIAN_FRONTEND=noninteractive sudo apt-get install -y --no-install-recommends \
+    ca-certificates tar g++ wget pkg-config nasm yasm libglib2.0-dev flex bison gobject-introspection libgirepository1.0-dev \
+    python3-dev libx11-dev libxv-dev libxt-dev libasound2-dev libpango1.0-dev libtheora-dev libvisual-0.4-dev libgl1-mesa-dev \
+    libcurl4-gnutls-dev librtmp-dev mjpegtools libx264-dev libx265-dev libde265-dev libva-dev libtbb-dev
+  
+  # Add commands to build FFMPEG from source
+  ffmpeg_dir=$base_dir/ivsr_ffmpeg_plugin/ffmpeg
+  ffmpeg_repo=https://github.com/FFmpeg/FFmpeg.git
+
+  if [ ! -d "${ffmpeg_dir}" ]; then
+    git clone --depth 1 --branch n6.1 ${ffmpeg_repo} ${ffmpeg_dir}
+    git config --global --add safe.directory ${ffmpeg_dir}
+  fi
+
+  # Apply patches
+  cd ${ffmpeg_dir} && cp -rf $base_dir/ivsr_ffmpeg_plugin/patches/*.patch .
+  git am --whitespace=fix *.patch
+
+  ./configure \
+      --enable-gpl \
+      --enable-nonfree \
+      --disable-static \
+      --disable-doc \
+      --enable-shared \
+      --enable-version3 \
+      --enable-libivsr \
```

Comment:
Not sure if it can find ivsr library when it builds FFmpeg. If not, we may consider adding "ldconfig" after line#100